File: cash6/cash.py
from cs50 import get_float


count = 0
while True:
    money = get_float("Input money: ")
    money = round(money * 100)
    if money > 0:
        break

while money > 0:
    if money - 25 >= 0:
        money = money - 25
        count += 1
    elif money - 10 >= 0:
        money = money - 10
        count += 1
    elif money - 5 >= 0:
        money = money - 5
        count += 1
    elif money - 1 >= 0:
        money = money - 1
        count += 1

print(count)

File: probleme_rez/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int count = 0;
    int money = get_int("Input money");

    while(money < 0)
    {
        money = get_int("Input money");
    }

    while(money > 0)
    {
        if(money - 25 >= 0)
        {
            money = money - 25;
            count ++;
        }else if(money - 10 >= 0)
        {
            money = money - 10;
            count ++;
        }else if(money - 5 >= 0)
        {
            money = money - 5;
            count ++;
        }else if(money - 1 >= 0)
        {
            money = money - 1;
            count ++;
        }
    }
    printf("%i\n", count);
}
